src/base/module/cursor/RecordCursor.js

```diff
@@ -50,11 +50,11 @@ class RecordCursor {
   }
   /**
    * Get 1 block of records
-   * @param {String} cursorID
+   * @param {String} id cursor id
```

Comment:
Update comment:
/**
  * request to URL
  * @param {Object} params
  * @param {String} params.cursorID
  * @return {Promise}
  */